api/CharacterData.json

```diff
@@ -0,0 +1,92 @@
+{
+  "api": {
+    "CharacterData": {
```

Comment:
The following sub-features are missing (see [DOM Standard](https://dom.spec.whatwg.org/#characterdata)):
* `appendData`
* `data`
* `deleteData`
* `insertData`
* `length`
* `replaceData`
* `substringData`

Since MDN doesn't have any compat data for these sub-features yet, we can set all browsers mentioned in this feature to `null`. I would also suggest adding `mdn_url`s, even if the sub-articles don't exist yet.